cognite/client/_api/sequences.py

```diff
@@ -730,7 +730,7 @@ def _convert_resource_to_patch_object(
         cdf_item_by_id: dict[str, Sequence] | None = None,  # type: ignore[override]
     ) -> dict[str, dict[str, dict]]:
         update_obj = super()._convert_resource_to_patch_object(resource, update_attributes, mode)
-        if not isinstance(resource, SequenceWrite):
+        if not isinstance(resource, SequenceWrite) or not resource.columns:
```

Comment:
...but if `mode=="replace"`, empty columns should mean remove, right?

Reply:
Good point